qa/suites/fs/thrash/multifs/yes/ceph-thrash/mon.yaml

```diff
@@ -0,0 +1,10 @@
+overrides:
+  ceph:
+    log-ignorelist:
+      - overall HEALTH_
+      - \(MON_DOWN\)
+tasks:
+- mon_thrash:
+    check_mds_failover: True
+    revive_delay: 20
+    thrash_delay: 10
```

Comment:
Do these need to be copies (`ceph-thrash`)?

Reply:
Done